d.ts/mpegts.d.ts

```diff
@@ -47,6 +47,11 @@ declare namespace Mpegts {
          * @defaultvalue false
          */
         enableWorker?: boolean;
+        /**
+         * @desc Enable Managed Media Source (unstable for now)
+         * @defaultvalue false
+         */
+        enableManagedMediaSource?: boolean;
```

Comment:
I don't think we need a config field to enable this. We should use it as long as it's detected.

Reply:
OK, i will fix ut,
if only ManagedMediaSource avaliable (iPhone), use ManagedMediaSource.

Reply:
Fixed!